Common/Data/UniverseSelection/CoarseFundamentalUniverse.cs

```diff
@@ -63,6 +78,24 @@ public CoarseFundamentalUniverse(Symbol symbol, UniverseSettings universeSetting
             _selector = selector;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CoarseFundamentalUniverse"/> class
+        /// </summary>
+        /// <param name="symbol">Defines the symbol to use for this universe</param>
+        /// <param name="universeSettings">The settings used for new subscriptions generated by this universe</param>
+        /// <param name="securityInitializer">Initializes securities when they're added to the universe</param>
+        /// <param name="selector">Returns the symbols that should be included in the universe</param>
+        public CoarseFundamentalUniverse(Symbol symbol, UniverseSettings universeSettings, ISecurityInitializer securityInitializer, PyObject selector)
```

Comment:
Do we want to add these ctor overloads to support python? Looks like the code that calls this is all custom code, is it possible for that code to perform the convert operation or does it have to happen in a c# context?

Reply:
I think that we do. If we want to create one of these universes (I added for `FineFundamentalFilteredUniverse` in #1979) directly from a python algorithm, the overload is there.